lib/Importer/ONNXModelLoader.cpp

```diff
@@ -1085,6 +1085,37 @@ ONNXModelLoader::loadConstantOfShape(const ONNX_NAMESPACE::NodeProto &op,
   return llvm::Error::success();
 }
 
+llvm::Error ONNXModelLoader::loadTile(const ONNX_NAMESPACE::NodeProto &op,
```

Comment:
C2 loader already have a logic for loading `Tile` ops. Have you considered merging those in CommonOperatorLoader?

Reply:
C2 tile has a different API than ONNX's. It takes number of repeats and Axis as arguments and only expands across 1 dimension. ONNX Tile does not take axis and takes repeats along each dimension. So I don't think we can make it in common code given different inputs semantics. :)

Reply:
fair enough